ci-operator/config/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-master.yaml

```diff
@@ -64,6 +64,11 @@ tests:
   commands: make ci-job
   container:
     from: src
+- as: ztp-only-ci
+  commands: make ztp-ci-job
+  container:
+    from: src
+  skip_if_only_changed: ^cnf-tests
```

Comment:
What do you mean by I can add another patterns?
What is missing?

Reply:
run on ztp
and skip on cnf-tests 
is the same logically,
and as far as I could see, couldn't find a pattern for: run only on or something like that.

Reply:
I think it's not the same, because you have a lot of directories in the repo root. You have `run_if_changed`. For example: https://github.com/openshift/release/blob/master/ci-operator/config/kubevirt/hyperconverged-cluster-operator/kubevirt-hyperconverged-cluster-operator-main__okd.yaml#L115

Reply:
oh I see what you mean

Reply:
thanks for the suggestion. I'll rework this

Reply:
Done. @sshnaidm